src/background.js

```diff
@@ -100,17 +113,36 @@ async function start() {
       requestHeaders,
     };
   }, { urls: [`${SERP_BASE_URL}/search*`]}, ["blocking", "requestHeaders"]);
+
+  browser.webRequest.onBeforeSendHeaders.addListener(async (details) => {
```

Comment:
Should this really be `async`?
```suggestion
  browser.webRequest.onBeforeSendHeaders.addListener((details) => {
```

Reply:
👍🏻